stdlib/universe/schema_functions_test.go

```diff
@@ -429,6 +585,91 @@ func TestDropRenameKeep_Process(t *testing.T) {
 				},
 			}},
 		},
+		{
+			name: "keep one key col merge tables",
+			spec: &universe.SchemaMutationProcedureSpec{
+				Mutations: []universe.SchemaMutation{
+					&universe.KeepOpSpec{
+						Columns: []string{"a", "c"},
+					},
+				},
+			},
+			data: []flux.Table{
+				&executetest.Table{
+					ColMeta: []flux.ColMeta{
+						{Label: "a", Type: flux.TString},
+						{Label: "b", Type: flux.TString},
+						{Label: "c", Type: flux.TFloat},
+					},
+					KeyCols: []string{"a", "b"},
+					Data: [][]interface{}{
+						{"one", "two", 3.0},
+						{"one", "two", 13.0},
+					},
+				},
+				&executetest.Table{
+					ColMeta: []flux.ColMeta{
+						{Label: "a", Type: flux.TString},
+						{Label: "b", Type: flux.TString},
+						{Label: "c", Type: flux.TFloat},
+					},
+					KeyCols: []string{"a", "b"},
+					Data: [][]interface{}{
+						{"one", "three", 5.0},
+						{"one", "three", 15.0},
+					},
+				},
+			},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "a", Type: flux.TString},
+					{Label: "c", Type: flux.TFloat},
+				},
+				KeyCols: []string{"a"},
+				Data: [][]interface{}{
+					{"one", 3.0},
+					{"one", 13.0},
+					{"one", 5.0},
+					{"one", 15.0},
+				},
+			}},
+		},
+		{
+			name: "keep one key col merge error column count",
```

Comment:
Should there be a test case for "keep one key col merge error column type"?

Reply:
I had omitted it because the `keep` path is pretty much the same as the drop one, but I added it for completeness.